docs/authentication/saml/overview.mdx

```diff
@@ -51,19 +51,15 @@ This will work: Once the user comes back from the IdP, they will need to go thro
 
 ### What happens if I delete the SAML connection? Will my users be deleted?
 
-The users will not be deleted, so your application will not break. However, they will need to “reintroduce” themselves to your new strategies by resetting their passwords or via OTP (depending on the strategy you choose).
+The users will not be deleted, so your application will not break. However, they will need to use another strategy (e.g. password, email OTP etc) to authenticate themselves moving forward.
 
 ### Does Clerk support IdP-initiated SSO?
 
-No, Clerk only supports SP-initiated flows at this time.
+Yes, Clerk supports both SP-initiated and IdP-initiated SSO flows.
 
-### How much does it cost now?
+### How much does it cost?
 
-It's going to be free during the Beta period for production instances. Connections in development instances will always be free, but capped to 5.
-
-### How much will it cost after Beta?
-
-It will cost $50 per active connection per month for production instances. Connections in development instances will always be free, but capped to 5.
+It costs $50 per active connection per month for production instances. Connections in development instances will always be free, but capped to 5.
```

Comment:
Maybe it's better to link users to our pricing page instead? Dunno if we are listing pricing options for any other feature as part of our docs

Reply:
Well here is what our pricing says:

<img width="841" alt="Screenshot 2024-04-01 at 13 12 47" src="https://github.com/clerk/clerk-docs/assets/98043211/4912aced-a7b7-4825-ac9a-e01433743320">

So $25 for pro, $100 add on to have SAML connections, and then this FAQ is saying another $50 per active connection per month (for prod instances)?? Where is the information for that?

Reply:
It's on the table below where we list the features per plan
![Screenshot 2024-04-02 at 11 07 40 AM](https://github.com/clerk/clerk-docs/assets/22435234/7ac54d07-6f0c-4190-8680-2c3cebfe134e)

Reply:
so realistically, the user is paying $25 for pro, $100 for enhanced auth add-on, and then $50 per saml connection. so if they have one saml connection, they're paying $175 a month - is this right ?
if it is, we can either 1. update the text to reflect that ^ or 2. simply link to the pricing page .... i'm not really sure how we should do it, i think marketing would have a better opinion.

@astrotars @domitriusclark @kylemac definitely missing some names, but they should be able to give us some feedback !

Reply:
Roy responded with this idea:

```suggestion
For production instances, each SAML connection will cost $50 per month. SAML connections require the Pro plan and the Enhanced Authentication Add-on. Please see [pricing](https://clerk.com/pricing) for those costs, and the costs of other add-ons or features.
For development instances, SAML connections will always be free, but capped to 5.
```

Reply:
Updated based on Roy suggestion